src/application/main.c

```diff
@@ -3,9 +3,11 @@
  *****************************************************************************/
 #include <stdbool.h>
 #include <stdint.h>
+#include <stdio.h>
```

Comment:
Nothing from `stdio.h` works at the moment, because `_write_r` and `_read_r` are stubs (see `src/system/stubs.c`).